.github/workflows/ContinuousIntegration.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        os: [ ubuntu-22.04, macos-12 ]
+        os: [ ubuntu-22.04, macos-13 ]
```

Comment:
do we need to change to macos-13 for all the others?

Reply:
I already did. This is in develop for all other libraries (NDItk, ACEtk, ENDFtk, scion, tools).